removing-members

From: Bryan Larsen <bryan@larsen.st>

## Removing members

The sidebar we just implemented has an obvious draw-back -- there's no way to remove members. In typical RESTful style, removing a member is achieved by deleting a membership. What we'd like is a delete button on each card that removes the membership. That means what we really want are *Membership* cards in the sidebar (at the moment they are User cards). So, in `projects/show.dryml`, change:

SHOW_PATCH
---
 app/views/projects/show.dryml |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/app/views/projects/show.dryml b/app/views/projects/show.dryml
index 1e9f38a..60b06d9 100644
--- a/app/views/projects/show.dryml
+++ b/app/views/projects/show.dryml
@@ -12,7 +12,7 @@
 
   <aside:>
     <h2>Project Members</h2>
-    <collection:members part="members"/>
+    <collection:memberships part="members"/>
 
     <form:memberships.new update="members" reset-form refocus-form>
       <div>
